Makefile

```diff
@@ -24,19 +14,17 @@ deps:
 		echo "ERROR: Cannot run on non-Linux systems"; \
 		false; \
 	fi
+	sudo apt-get remove docker docker-engine
```

Comment:
in the travis. We don't want to remove a docker installation on the user's machine. if there's already a docker, it will just upgrade it.